sample/src/main/java/com/google/accompanist/sample/swiperefresh/DocsSamples.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.sample.swiperefresh
+
+import androidx.compose.runtime.Composable
+
+@Composable
+fun SwipeRefreshSample() {
+    // TODO
```

Comment:
Should we add this in? Could just copy `SwipeRefreshBasicSample`

Reply:
Good catch, done.